__init__.py

```diff
@@ -184,19 +184,20 @@ def rate_on_ankiweb(did=None):
 
 
 def visualize_schedule(did=None):
+    url = "https://open-spaced-repetition.github.io/anki_fsrs_visualizer"
     deck = mw.col.decks.current()
-    config = mw.col.decks.get_config(deck["conf"])
-    retention = config["desiredRetention"]
-    fsrs_params = (
-        config["fsrsParams5"]
-        if "fsrsParams5" in config and len(config["fsrsParams5"]) > 0
-        else config["fsrsWeights"]
-    )
-    fsrs_params_string = ",".join(f"{x:.4f}" for x in fsrs_params)
-
-    openLink(
-        f"https://open-spaced-repetition.github.io/anki_fsrs_visualizer/?w={fsrs_params_string}&m={retention}"
-    )
+    if "config" in deck:
```

Comment:
```suggestion
    if "conf" in deck:
```
![image](https://github.com/user-attachments/assets/ba066d33-599e-4b64-8e0a-da714197613b)
The filtered deck has no conf.